desktop-environment.el

```diff
@@ -176,6 +193,17 @@ portion of the screen."
   "Shell command toggling bluetooth."
   :type 'string)
 
+
+;;; Helper variables - keyboard layout
+
+(defvar desktop-environment-keyboard-layouts-ring []
+  "Varible for storing whole commands for setxkbmap.
+Do not edit this by hand! Use desktop-environment-keyboard-layouts variable instead.")
+
+(defvar desktop-environment-current-keyboard-layout-index -1
```

Comment:
please make the variable private by using 2 dashes: `desktop-environment--keyboard-layouts-index`